src/commands/run.js

```diff
@@ -55,17 +62,20 @@ class RunCommand extends ApifyCommand {
         // NOTE: User can overwrite env vars
         const env = Object.assign(localEnvVars, process.env);
 
-        await execWithLog('node', [MAIN_FILE], { env });
+        await execWithLog('npm', ['start'], { env });
     }
 }
 
 // TODO: we should describe which env vars are set here:
 
-RunCommand.description = 'Runs the actor locally in the current directory.\n' +
-    'The command runs a Node.js process with the actor in the current directory. It sets various APIFY_XYZ environment variables ' +
-    'in order to provide a working execution environment for the actor. For example, this causes ' +
-    'the actor input, as well as all other data in key-value stores, datasets or request queues to be stored in the "apify_local" directory, ' +
-    'rather than on the Apify platform.';
+RunCommand.description = 'Runs the actor locally in the current directory.\n'
+    + 'The command runs a npm start script in the current directory. It sets various APIFY_XYZ environment variables '
```

Comment:
Runs the actor locally in the current directory by executing "npm start". The command sets various APIFY_XYZ ...